lib/DoctrineCodingStandard/Helpers/UseStatementHelper.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineCodingStandard\Helpers;
+
+use PHP_CodeSniffer\Files\File;
+use SlevomatCodingStandard\Helpers\UseStatementHelper as UseHelper;
+use Throwable;
+use function in_array;
+
+class UseStatementHelper
+{
+    /**
+     * @return string[]
+     */
+    public static function getUseStatements(File $phpcsFile) : array
+    {
+        $importedClasses = [];
+        foreach (UseHelper::getUseStatements($phpcsFile, 0) as $useStatement) {
+            $importedClasses[$useStatement->getNameAsReferencedInFile()] = $useStatement->getFullyQualifiedTypeName();
+        }
+
+        return $importedClasses;
+    }
+
+    /**
+     * @param string[] $importedClassNames
+     * @param string[] $implementedInterfaces
+     */
+    public static function isImplementingThrowable(array $importedClassNames, array $implementedInterfaces) : bool
+    {
+        return (in_array(Throwable::class, $importedClassNames, true) &&
+            in_array(Throwable::class, $implementedInterfaces, true)) ||
```

Comment:
Just to make sure, I assume this could lead to issues if `Throwable` is aliased?

```
use My\Namespace\Throwable;
use Throwable as BaseThrowable;

class Exception implements Throwable
```

We might want to also fetch aliases from `use` statements and explicitly check against that to avoid wrong results.

Reply:
Like this one?
https://github.com/doctrine/coding-standard/pull/63/files#diff-fea81a7c92d7e638eff90b9ea3554855R36

Reply:
I guess the whole thing depends on whether `findImplementedInterfaceNames` returns aliased names or the actual FQCN. If it's the latter, this works as expected.

FWIW, there's a typo in `TrowableAlias` in your example that I just spotted now.

Reply:
Can you please point to the typo?